docs/arcaflow/workflows/schemas.md

```diff
@@ -39,18 +41,18 @@ Schemas that are not composed within an ObjectSchema do not have an `Object ID`.
 |---------------------------|---------------|-------------------------------|
 | Object                    | MyFirstObject | MyFirstObject                 |
 | List[Object]              | MyFirstObject | list_MyFirstObject            |
-| List[List[String]]        | MyFirstObject | list_list_MyFirstObject       |
-| List[List[List[String]]]  | MyFirstObject | list_list_list_MyFirstObject  |
+| List[List[Object]]        | MyFirstObject | list_list_MyFirstObject       |
+| List[List[List[Object]]]  | MyFirstObject | list_list_list_MyFirstObject  |
  
 ### Remaining Schemas
 

```

Comment:
I suggest "Other Schemas" rather than "remaining":  the reader is unlikely to know which ones are "left", so "remaining" doesn't really fit the situation.

Reply:
changed